apps/web/src/design-system/button/Button.tsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import { Button as MantineButton, SharedButtonProps } from '@mantine/core';
+
 import useStyles from './Button.styles';
+
 import { SpacingProps } from '../shared/spacing.props';
 
+export type Size = 'md' | 'lg' | undefined;
```

Comment:
I noticed there are not many shared typings in the Web app. So I left it here until the creation of typings is more common. I neither implement it in other places where size are typed hardcoded.